safe-global/safe-wallet-web
# feat: unsupported chain disconnection feedback

## What it solves

Resolves no feedback when automatically disconnected from a dApp when switching to an unsupported chain

## How this PR fixes it

When a session is deleted, if it is because of an unsupported chain, a new notification popup is shown.

## How to test it

Open a dApp on a chain that is supported and switch to one that is not. Observe the new popup.

## Screenshots

![disconnection-feedback](https://github.com/safe-global/safe-wallet-web/assets/20442784/8f528e16-a35d-4b1c-9fc9-ff9303ca5b35)

## Checklist
